### DISCUSSION TITLE
Welcome to pdftron-sign-app Discussions!

### DISCUSSION THREAD

**Discussion by andreysaf** [PDFTron/pdftron-sign-app#6] [General]
## PDFTron Sign App

Here you can:
- Post some ideas on how the app can be improved
- What features are missing

**Comment by leonardooliveira64** [comment]
I would like more information about licensing, what is the cost? how it works?

**Reply by andreysaf** [reply]
Hey @leonardooliveira64, you can just fill out a form here and someone will be with you in touch: https://www.pdftron.com/form/contact-sales/.
